.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,72 @@
+name: Bug report
+description: You found a bug or encountered a crash? Please report it here.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Provide a summary of the issue in the Title above, please do not "[Tag]" it.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: |
+        Please describe as best as possible.
+        Always use the most recent version from [our releases](https://github.com/AppliedEnergistics/Applied-Energistics-2/releases) or curseforge, it might already be fixed.
```

Comment:
Why are you not linking to curseforge here? We don't publish binaries on the release-page

Reply:
We actually do and it's pretty much guaranteed to be the most recent version. Curseforge is not under our control, so in case they decide to wait 2 days for their review, reject it for some reason, or it's an alpha which isn't shown by default, there might be discrepancy.

Reply:
WTF, we actually do. I take it back :-P